examples/gno.land/r/gnoland/events/errors.gno

```diff
@@ -0,0 +1,9 @@
+package events
+
+import "errors"
+
+var (
+	errEmptyName        = errors.New("event name cannot be empty")
```

Comment:
```suggestion
	ErrEmptyName        = errors.New("event name cannot be empty")
```

so it can be used by external packages to compare an error.

Reply:
[9a9e215](https://github.com/gnolang/gno/pull/2372/commits/9a9e215feb07d596c0d5f85e6a91f0ab97728ef3)